test/SConsignFile/default.py

```diff
@@ -20,12 +22,9 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-#
-
-__revision__ = "__FILE__ __REVISION__ __DATE__ __DEVELOPER__"
 
 """
-Verify the default behavior of SConsignFile(), called with no arguments.
+Verify the behavior of SConsignFile() called with a subst-able path.
```

Comment:
There's no subst'ing here?

Reply:
Fixed - subst comment went onto the wrong file.